controllers/generateUniqueToken.js

```diff
@@ -0,0 +1,16 @@
+const uuid = require("uuid");
+
+/**
+ * Route used to generate a unique token
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const generateToken = (req, res) => {
+  const token = uuid.v4();
```

Comment:
do we have the uuid package already installed?

Reply:
Response screenshot added @shreya-mishra

Reply:
> do we have the uuid package already installed?

No @shreya-mishra , it was not there, i have added it now.